src/renderer/ducks/deposit/actions.ts

```diff
@@ -0,0 +1,15 @@
+import {depositSlice} from "./slice";
+import {createAction} from "@reduxjs/toolkit";
+import {INetworkConfig} from "../../services/interfaces";
+
+export const {
+    storeDepositTx, depositDetected, depositNotFount, waitForDeposit, resetDepositData
+} = depositSlice.actions;
+
+export const generateDeposit = createAction<INetworkConfig>("deposit/generateDeposit");
+
+type VerifyDeposit = (networkConfig: INetworkConfig, timeout?: number) =>
+{payload: {networkConfig: INetworkConfig; timeout: number}};
+export const verifyDeposit = createAction<VerifyDeposit>(
```

Comment:
same here with inline, it's hard to read. maybe we can add an eslint rule for this :)

Reply:
best solution is to implement `prettier`